packages/autocomplete-core/rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import resolve from '@rollup/plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import replace from '@rollup/plugin-replace';
+import { name } from './package.json';
+
+const createPlugins = ({ format }) => [
+  replace({
+    __DEV__: format === 'umd' ? false : 'process.env.NODE_ENV !== "production"',
```

Comment:
I think that this is enough:

```suggestion
    __DEV__: JSON.stringify(process.env.NODE_ENV === 'development'),
```

Reply:
https://github.com/francoischalifour/autocomplete.js/pull/26/commits/44ad404d1025617befaa878e4ebe0a22b7e1421b